src/sql/workbench/contrib/views/browser/treeView.ts

```diff
@@ -670,6 +670,7 @@ class TreeDataSource implements IAsyncDataSource<ITreeItem, ITreeItem> {
 		private treeView: ITreeView,
 		private id: string,
 		private withProgress: <T>(task: Promise<T>) => Promise<T>,
+		@IConfigurationService private readonly configurationService: IConfigurationService,
```

Comment:
the configurationService is what holds our user configuration.  We need to add this service to some classes, and add it as a parameter to some methods.